README.md

```diff
@@ -47,6 +47,24 @@ Put your custom node scripts into a folder called `node` in the root and add sub
 Use `yarn` to install dependencies
 ```yarn install```
 
+## Release
+In order to release a new version of the project.
+
+```yarn release```
+
+A new version will be calculated based on your commit history – make sure to follow [conventional commit message guidelines](https://github.com/conventional-changelog/standard-version#commit-message-convention-at-a-glance).
+Additionally the CHANGELOG.md will be updated automatically.
+
+## Pre-Release
+
+```yarn release -- --prerelease```
```

Comment:
Without explanation could be confusing, since this is less common, perhaps we could skip it and the user can look it up him/herself